src/routes/(auth)/acceptprojectinvitation/+page.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+    import { goto } from "$app/navigation";
+    import { Button } from "$lib/components/primitives/button";
+    import type { RpcError } from "@protobuf-ts/runtime-rpc";
+    import { GrpcStatusCode } from "@protobuf-ts/grpcweb-transport";
+    import { getGrpcStatusCode } from "$lib/utils/common-helper";
+
+    let { data } = $props();
+    const { acceptancePromise: acceptancePromise } = data;
```

Comment:
```suggestion
    const { acceptancePromise } = data;
```
Why not? You are not renaming it?!